addons/account/static/src/components/systray_item/systray_item.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates xml:space="preserve">
+    <t t-name="account.SystrayItem" owl="1">
+        <div t-if="is_quick_edit_mode_enabled">
+            <a title="The name of the company is displayed because there is a single company and the accounting firm mode is on." class="text-white" t-esc="currentCompany.name"/>
```

Comment:
You don't have to explain in the UI why this or that is displayed :sweat_smile: 
```suggestion
            <a class="text-white" t-esc="currentCompany.name"/>
```

Also, why the `text-white` class? Isn't the color supposed to be set by `o_main_navbar something`? (you'll notice that the color is not exactly the same as there is some alpha value)